toolchain/check/context.cpp

```diff
@@ -618,6 +618,28 @@ auto Context::ParamOrArgEnd(Parse::NodeKind start_kind) -> SemIR::InstBlockId {
   return ParamOrArgPop();
 }
 
+auto Context::FinalizeInits() -> void {
+  inst_block_stack().PushInit();
+  if (!inst_block_stack().PeekCurrentBlockContents().empty()) {
+    AddInst({Parse::NodeId::Invalid, SemIR::Return{}});
+    // Pop the Inits block here to finalize it
+    inst_block_stack().Pop();
+
+    // __global_init is only added if there are initialization instructions
+    auto nameid = sem_ir().identifiers().Add("__global_init");
+    sem_ir().functions().Add(
+        {.name_id = SemIR::NameId::ForIdentifier(nameid),
```

Comment:
```suggestion
    auto name_id = sem_ir().identifiers().Add("__global_init");
    sem_ir().functions().Add(
        {.name_id = SemIR::NameId::ForIdentifier(name_id),
```

Style: underscores are used to separate words in [variable names](https://google.github.io/styleguide/cppguide.html#Variable_Names).